app/views/preferences/index.html.erb

```diff
@@ -1,160 +1,160 @@
 <!--Descriptive page name, messages and instructions-->
-<h2 class="heading"><%= ts('Set My Preferences') %></h2>
+<h2 class="heading"><%= t(".page_heading") %></h2>
 <%= error_messages_for :preference %>
 
 <!--/descriptions-->
 
 <!--subnav-->
 <h3 class="landmark heading">Navigation</h3>
```

Comment:
Sorry if I'm wrong, but shouldn't this also be i18n-ed?

Reply:
Indeed it should, thanks!